renku/core/management/storage.py

```diff
@@ -432,27 +431,34 @@ def get_lfs_migrate_filters(self):
 
             if p.include:  # File ignored by LFS
                 excludes.append(pattern)
+                if pattern not in self.RENKU_PROTECTED_PATHS:
+                    excludes.append(f"{pattern.strip()}/**")
```

Comment:
Nice solution, I didn't think of just adding the pattern with/without `/**`.

I guess if a user does `myfolder/*`, explicitly intending to only ignore top-level files in the directory, this would break that? But then I think we can live with that.

```suggestion
                    excludes.append(f"{pattern.strip().rstrip("/")}/**")
```
maybe? I'm not sure what happens if my patterns is `data/` and it turns to `data//**`?

Reply:
> I guess if a user does myfolder/*, explicitly intending to only ignore top-level files in the directory, this would break that? But then I think we can live with that.
Good point! I've added a check for it.